examples/pytorch/vision/README.md

```diff
@@ -0,0 +1,131 @@
+<!---
```

Comment:
@sgugger - is `pytorch/vision` not a bit too general? Given that we have `examples/pytorch/text-classification/run_glue.py` I think the better structure would be `examples/pytorch/image-classification/run_image_classification.py` here no?

Reply:
Never mind - just saw your comment!